presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -55,6 +63,18 @@ public static Slice ObjectId(@SqlType(StandardTypes.VARCHAR) Slice value)
         return Slices.wrappedBuffer(new ObjectId(CharMatcher.is(' ').removeFrom(value.toStringUtf8())).toByteArray());
     }
 
+    @ScalarOperator(CAST)
+    @LiteralParameters("x")
+    @SqlType("varchar(x)")
+    public static Slice castToVarchar(@LiteralParameter("x") long x, @SqlType("ObjectId") Slice value)
```

Comment:
Maybe call the argument `length`?

Reply:
we often call it just `x`, eg. https://github.com/prestosql/presto/blob/4b6a52d80bb4e8c81acaef1b0b61725ac10c7edd/presto-main/src/main/java/io/prestosql/operator/scalar/CharacterStringCasts.java#L47
then there is no possibility to infer what "length" it is